bin/typescript-node-petstore-with-npm.sh

```diff
@@ -27,6 +27,6 @@ fi
 
 # if you've executed sbt assembly previously it will use that instead.
 export JAVA_OPTS="${JAVA_OPTS} -XX:MaxPermSize=256M -Xmx1024M -DloggerPath=conf/log4j.properties"
-ags="generate -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -g typescript-node -c bin/typescript-petstore-npm.json -o samples/client/petstore/typescript-node/npm $@"
+ags="generate -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -g typescript-node -c bin/typescript-node-petstore-npm.json -o samples/client/petstore/typescript-node/npm $@"
```

Comment:
windows scripts needs to be adapted, too:
https://github.com/OpenAPITools/openapi-generator/blob/2162d34ed78ccdcd7ae30dad0483dd27e36d798b/bin/windows/typescript-node-petstore-with-npm.bat#L8